public/content/lessons/2019/windmills/index.mdx

```diff
@@ -1,9 +1,232 @@
+
+In our example, one way to formalize the idea of a “middle” is to count how many points are on either side of the line. If you give the line some orientation, you can reasonably talk about the left half, say coloring all points to the left of it blue, and the right half, coloring all points to the right of it brown, and what it means for the line to be in the middle is that there are as many blue points and brown points.
+
+For the moment, let’s say the total number of points is odd, and the point the line passes through is colored white. So for example, if there were 11 points, then you’d have five blue ones on the left of the line, and five brown ones on the right of the line, and the one white one at the pivot. The case with an even number of points will be similar, just slightly less symmetric.
+
+<Figure
+    image="introduce_colors.png"
+/>
+
+This gives you a new question to ask: What happens to the number of blue points and brown points as the process plays out? In the example shown, you might notice that it’s always 5 and 5, never changing. Playing with other examples, you’d find the same is true. Take a moment to pause and see if you can think about why that would be true.
+
+<Figure
+    image="constant.png"
+    video="constant.mp4"
+    show="video"
+/>
+
+Well, the key is to think through what happens as a line changes its pivot. Having given the line an orientation, we can talk about which half of the line is “above” the pivot, and which is “below”. If the line hits a blue point to its left, it must happen below the pivot, so as it changes pivots and continues rotating clockwise, the old pivot, now above the new one, ends up to the left of the line, in the blue region. And entirely symmetrically, when it hits a brown point, it happens above the pivot, meaning that old pivot ends up in the brown region. So no matter what, the number of points on a given side of the line can’t change. When you lose a blue point, you gain a new one; when you lose a brown point, you also gain a new one.
+
+<Figure
+    image="change_pivot.png"
+    video="change_pivot.mp4"
+/>
+
+Great, that’s insight #1. Now why would this imply that the line must hit every point infinitely many times, no matter what weird set of points you dream up? Well, think about letting this process go until the line has turned 180 degrees. That means it’s parallel to its starting position, and because it has to remain the case that half the points are on one side, half are on the other, it must be passing through the same point it started on. Otherwise, if it ended up on some other point, it would change the number on a given side. Additionally, since the line has rotated halfway around, everything that was blue has become brown, and everything which was brown has become blue, and the only way to change color is if you get hit by the line.
```

Comment:
```suggestion
Great, that’s insight #1. Now why would this imply that the line must hit every point infinitely many times, no matter what weird set of points you dream up? Well, think about letting this process go until the line has turned $180^\circ$. That means it’s parallel to its starting position, and because it has to remain the case that half the points are on one side, half are on the other, it must be passing through the same point it started on. Otherwise, if it ended up on some other point, it would change the number on a given side. Additionally, since the line has rotated halfway around, everything that was blue has become brown, and everything which was brown has become blue, and the only way to change color is if you get hit by the line.
```

Reply:
This is also a point where a question could be inserted.

<Question
  question="If the line has rotated $180^\circ$, and ends up passing through the same point where it began, has it necessarily touched every point along the way?"
  choice1="Yes"
  choice2="No"
  answer={1}
>

Since the line has rotated halfway around, every point which was initially to the line's left, and so was colored blue, is no to the line's right, and so is colored brown. Likewise, every point which began brown becomes blue. The only way to change color is if you get hit by the line.

</Question>